templates/base.html

```diff
@@ -76,139 +75,131 @@
     <body class="{% if profile %}{% if is_external_student %}external-student{% else %}internal-student{% endif %}{% endif %} lang-{{ request.LANGUAGE_CODE }}" data-view-tag="{% block view_tag %}{% endblock %}">
         <a href="#content" class="skip-link">{% trans "Skip main navigation" %}</a>
         <div class="page-wrap">
-            <header>
```

Comment:
These html changes are not documented (i.e. they are just part of the sass commit). Do I read correctly, that you just removed `<header>` and changed `<footer>` to `<nav>`? Where those part of some other PR too, or do I just remember wrong?

Reply:
You're quite right! I'd missed these, they were previously introduced in the landmarks PR, and I was using an old version of `base.html` here. 

I've updated it now, so the only changes to this file should be removing the CDN Bootstrap CSS, which is part of the SASS changes as the SASS version of the CSS will include all of the Bootstrap styles.